### DISCUSSION TITLE
How to cite the MBC?

### DISCUSSION THREAD

**Discussion by RazviOverflow** [MBCProject/mbc-markdown#128] [General]
I was wondering how to correctly cite the Malware Behavior Catalog. I think there must be a standard way of referencing it.

With that idea in mind I PRed https://github.com/MBCProject/mbc-markdown/pull/127.

What do you think?

**Comment by dzbeck** [comment]
Thanks, @RazviOverflow! creating a cff file is a great idea. I just suggested a couple of edits to your PR. Please let me know what you think.

**Reply by dzbeck** [reply]
p.s. sorry for the delay - I've been out of the office.

**Reply by RazviOverflow** [reply]
No worries @dzbeck. I've checked the edits, they look good to me.

**Comment by dzbeck** [comment]
#127 has been merged.
